kmir/src/tests/integration/conftest.py

```diff
@@ -0,0 +1,3 @@
+# autoflake: skip_file
+
+from kmir.testing.fixtures import kmir, tools
```

Comment:
I'm not sure about this @tothtamas28 . I couldn't find another way to get pytest to find/use the fixtures that are defined in `kmir.testing.fixtures`.